_events/sr2017/southampton-tech-day-december.md

```diff
@@ -12,7 +12,14 @@ The Southampton tech days take advantage of the facilities of the University of
 
 ## Directions
 
-This years tech day will be at the University of Southampton, the exact location will be announced soon!
+The location of our tech day is on a different campus to that of Kickstart, please check our map!
+
+* How to [get to campus][soton-campus-directions]
+
+* [On-campus directions][soton-directions]
+
+* [Where to find food][soton-food-map]. Food is scarce in the Tech day location, so we recommend either ordering food in or taking packed lunches.
```

Comment:
This list renders borked for me. @jimmythompson please can we get #42 shipped ASAP since these pages look terrible without it.

@Adimote you can mitigate some of the borkage by removing the empty lines between the bullet points (which is what markdown expects btw).